Makefile

```diff
@@ -51,6 +51,15 @@ ifneq ($(OVERRIDE_IMAGE_NAME),)
 	docker tag $(PREFIX)/$(DOCKER_IMAGE):$(VERSION) $(OVERRIDE_IMAGE_NAME)
 endif
 
+container_rhel: build
```

Comment:
Please add a comment explaining this needs to be run on a RHEL box with podman available

Reply:
Done..